csharp/src/Microsoft.ML.OnnxRuntime/NativeMethods.cs

```diff
@@ -664,6 +671,25 @@ static NativeMethods()
         public delegate IntPtr /*(OrtStatus*)*/DOrtAllocatorGetInfo(IntPtr /*(const OrtAllocator*)*/ ptr, out IntPtr /*(const struct OrtMemoryInfo**)*/info);
         public static DOrtAllocatorGetInfo OrtAllocatorGetInfo;
 
+        /// <summary>
+        /// Create an instance of arena configuration which will be used to create an arena based allocator
+        /// </summary>
+        /// <param name="maxMemory">TODO</param>
```

Comment:
The original PR states that these parameters will be documented in OnnxRuntime_Perf_Test.md - @pranavsharma is going to add them, and once that is added I will use similar wording here to keep it consistent

Reply:
Added here https://github.com/microsoft/onnxruntime/pull/5831